.eslintrc.js

```diff
@@ -18,8 +20,9 @@ module.exports = {
   ],
   rules: {
     'react/no-set-state': 'off',
-    'max-lines-per-function': [1, 40],
+    'max-lines-per-function': [1, 80],
```

Comment:
120 is standard

Reply:
sorry, 120 is a standard when it comes to signs per line
80 is OK when it comes to function length